.contrib/Parser/DATAS/03 - World Drops/06 Cataclysm/03 Weapons.lua

```diff
@@ -185,6 +185,9 @@ _.WorldDrops =
 				i(66940),	-- Staff of Solemn Secrecy
 				i(55336),	-- Steelspark Staff
 				i(55444),	-- Steelspark Staff
+				i(62072, {	-- Robble's Wobbly Staff
+						["description"] = "Generic Cataclysm drop.",
+				}),
```

Comment:
Don't think the description is needed here, they are all generic drops.
```suggestion
				i(62072), -- Robble's Wobbly Staff
```